qemu/tests/cfg/multi_vms_with_stress.cfg

```diff
@@ -15,9 +15,11 @@
     pre_command = "mkdir -p ${stress_inst_dir}"
     post_command = "killall -9 stress; rm -rf ${stress_inst_dir}"
     ovmf:
```

Comment:
It will hit high failed ratio due to low performance host, just suggest using 3 vms for all platform.

vms += " vm2 vm3 "

LGTM

Reply:
Corrected